### DISCUSSION TITLE
Unstar deleted repos

### DISCUSSION THREAD

**Discussion by ivankovnatsky** [community/community#24756] [Apps, API and Webhooks]
<p>my page: <a href="https://github.com/ivankovnatsky?tab=stars" class="inline-onebox" rel="noopener nofollow ugc">ivankovnatsky (Ivan Kovnatsky) / Starred · GitHub</a></p>
<p>shows that i have 4 starred repos, but lists only 2:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/8/2823b816f519865a61885bd986672eadf3955d19.png" data-download-href="/uploads/short-url/5J5CQNf2084hzYjeNTEbJ7ANahX.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75213/181086587-97f25c2c-d9e8-432e-99d6-eab176346eaa.png" alt="image" data-base62-sha1="5J5CQNf2084hzYjeNTEbJ7ANahX" width="690" height="282" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/8/2823b816f519865a61885bd986672eadf3955d19_2_690x282.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/8/2823b816f519865a61885bd986672eadf3955d19_2_1035x423.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/8/2823b816f519865a61885bd986672eadf3955d19_2_1380x564.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/8/2823b816f519865a61885bd986672eadf3955d19_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1898×778 74.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>using api i was able to get the list of repos that i starred:</p>
<pre><code class="lang-auto">    "full_name": "kanashimia/nixos",
    "full_name": "cole-mickens/nixcfg",
    "full_name": "TechBookHunter/Free-Career-Guide-Books",
    "full_name": "TechBookHunter/Free-DevOps-Books",
</code></pre>
<p>first two the legitimate stars, while the other two are deleted repos i presume.</p>
<p>any way to unstar this?</p>

**Comment by canuckjacq** [comment]
<p>I’m betting from the repository names that they were subject to a <a href="https://github.com/github/dmca">DMCA takedown request</a>, so they may still exist, but they are disabled.</p>
<p>Since you can list these on the API, can you also <a href="https://docs.github.com/en/rest/reference/activity#unstar-a-repository-for-the-authenticated-user">unstar them</a>?</p>
<pre><code class="lang-auto">curl \
  -X DELETE \
  -H "Accept: application/vnd.github.v3+json" \
  https://api.github.com/ivankovnatsky/starred/TechBookHunter/Free-Career-Guide-Books
</code></pre>

**Comment by ivankovnatsky** [comment]
<p>i’m getting 404:</p>
<pre><code class="lang-auto">* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x7fe35c808200)
&gt; DELETE /ivankovnatsky/starred/TechBookHunter/Free-Career-Guide-Books HTTP/2
&gt; Host: api.github.com
&gt; User-Agent: curl/7.64.1
&gt; Accept: application/vnd.github.v3+json
&gt;
* Connection state changed (MAX_CONCURRENT_STREAMS == 100)!
&lt; HTTP/2 404
&lt; server: GitHub.com
&lt; date: Mon, 17 May 2021 19:25:05 GMT
&lt; content-type: application/json; charset=utf-8
&lt; content-length: 84
&lt; x-github-media-type: github.v3; format=json
&lt; x-ratelimit-limit: 60
</code></pre>

**Comment by canuckjacq** [comment]
<p>Thanks for trying - I’ve asked our engineers to see if they can sort this out for you! I’ll let you know when I hear back :slight_smile:</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>That should be done for you now! Sorry about the hassle.</p>

**Comment by ivankovnatsky** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a> indeed, old stars gone, awesome, thanks.</p>
